code-quality/test-files-indentation.s.md

```diff
@@ -0,0 +1,70 @@
+# [Standard] Test files indentation
+
+## Owner: [Pierre-Louis Le Portz](https://github.com/pleportz)
+
+## Why
+
+Standardizing the indentation of a test file saves the developer 30 seconds per new function to test.
+
+## Checks
+
+The test file must have the following indentation even if only one function is tested:
```

Comment:
You're giving one example but no clear rule of what it should be.

Can you describe the indentation in a few sentences that can be checked?